vendor/bat-native-ads/src/bat/ads/internal/ad_events/ad_event_util.h

```diff
@@ -8,12 +8,29 @@
 
 #include "bat/ads/internal/ad_events/ad_event_info_aliases.h"
 
+namespace absl {
```

Comment:
I would prefer not to do forward declaration of absl::optional as it will be converted to std::optional

Reply:
Raised https://github.com/brave/brave-browser/issues/18309, thanks